### ISSUE TITLE
Add pause functionality

### ISSUE THREAD

**Issue by drgn-hart** [drgn-hart/Countdown-Timer#14]
Currently the timer lets you start and stop. There is no way to pause the time. We need to be able to pause the time and then resume from the paused time.

**Comment by yuanLeeMidori** [comment]
Hi! I'm interested in working on this feature. Do you have instructions on how to contribute to your repo? Such as the required environment and the set-ups for running the application, also, what should I keep in mind while doing the contribution. Thank you!

**Comment by drgn-hart** [comment]
Hi @yuanLeeMidori ,
Thanks for your interest. I am still working on creating a Contributing.md file for this project. As of now, from setup perspective it is similar to any standard chrome extension setup and please make sure that eslint passes on the code you write.
If you can add unit tests for the changes that would be great!! As of now, I have just added a sample unit test.
Please feel free to ask if you have any questions.

**Comment by yuanLeeMidori** [comment]
Awesome! I'll finish the pause feature first and try to make the unit test. 

Just one question, there are currently 2 buttons which are "Start" and "Stop". I'll add a "Resume" for continuing the counting. However, since the current "Stop" works like pause functionality, do you want another "Pause" button, or "Resume" is just fine? 

Thank you :)

**Comment by drgn-hart** [comment]
Below is the plan I have for this extension:
User sets a time on the UI and clicks **start.** This will add the timer bar on the top of the web page he is currently on. Whenever user wants to get rid of the timer he will click on **stop**. This will reset the timer as well as remove it from UI. "Pause" and "Resume" feature will come in between this. When user clicks "Pause", timer bar will still remain on the page however timer will not be running, and user can use "Resume" button to resume the timer from where he has paused.
I am still thinking on should we show all 4 buttons all the time or should we display the buttons conditionally and I am open to suggestions on this. I believe this can be picked up in as different task by itself.
So, in short I was thinking of 4 different buttons for Start, Stop, Pause and Resume.
